src/components/ApplicationsTable/ApplicationsTable.tsx

```diff
@@ -17,9 +17,9 @@ type ApplicationsTableRowProps = {
 
 const ApplicationsTableRow = ({
   bookmark,
-  userCollectionOptions = [],
+  userCollectionOptions,
   handleAddToCollection,
-  canAddNewCollection = true,
```

Comment:
I was hunting for ways to bump test coverage and stumbled upon this. I removed these default params because they were never being evaluated due to the parent component having the same props with the same default param values.